test/suite/test_backup29.py

```diff
@@ -59,22 +59,37 @@ def parse_blkmods(self, uri):
         self.pr("block bitmap: " + blocks)
         return blocks
 
+    def compare_bitmap(self, orig, new):
+        # Compare the bitmaps from the metadata. Once a bit is set, it should never
+        # be cleared. But new bits could be set. So the check is only: if the original
+        # bitmap has a bit set then the current bitmap must be set for that bit also. 
+        #
+        # First convert both bitmaps to a binary string, accounting for any possible leading
+        # zeroes (that would be truncated off). Then compare bit by bit.
+        orig_bits = bin(int('1'+orig, 16))[3:]
+        new_bits = bin(int('1'+new, 16))[3:]
+        self.pr("Original bitmap in binary: " + orig_bits)
+        self.pr("Reopened bitmap in binary: " + new_bits)
+        for o, n in zip(orig_bits, new_bits):
```

Comment:
I would recommend using something other than `o` as a variable name, as it looks perhaps too much like 0.